sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanExec.scala

```diff
@@ -133,7 +133,7 @@ case class AdaptiveSparkPlanExec(
       inputPlan, queryStagePreparationRules, Some((planChangeLogger, "AQE Preparations")))
   }
 
-  @volatile private var currentPhysicalPlan = initialPlan
+  @volatile var currentPhysicalPlan = initialPlan
```

Comment:
Exposing a mutable variable seems not a good idea.

Reply:
Yes. Fixed.